documentation/faq.mdx

```diff
@@ -69,29 +69,57 @@ No, these games run on completely different engines and were not made by Naughty
 
 No. _Ratchet & Clank_ does not use Naughty Dog's engine and was not written in GOAL, it was developed by a completely different team of people. As stated in staff roll for the _Ratchet & Clank_ games, only the background renderers use Naughty Dog technology. Our work is not transferrable to that series.
 
+### Will you decompile The Last Of Us? Isn't it written in GOAL?
+
+To our konwledge, the four Jak games are the only ones ever written using GOAL. Naughty Dog did demonstrate how they use LISP in The Last Of Us.
+
 ### Will you decompile [_insert non-Jak game here_]?
 
 No, we are no where near done with Jak yet, and other than experience not much is transferable from this project to others. The Jak games present a very unique set of challenges that we have solved that other games do not -- similarly other games pose entirely new sets of challenges.
+<!-- https://discord.com/channels/756287461377703987/801612566874554399/1221207684074963014 -->
 
 Most importantly, we are motivated on working on Jak, not other games. Those other games likely already have small niche communities plugging away at things in the background just like we were.
 
 ### Is there support for raytracing?
 
-Implementing raytracing would require a very massive art overhaul which is far outside the scope of this project.
+No. Implementing raytracing would require a very massive art overhaul which is far outside the scope of this project.
 
 ## Game Info
 
+### What versions are supported?
+
+All of them! For both _Jak and Daxter: The Precursor Legacy_ and _Jak II_.
```

Comment:
This should probably be rephrased either way. It should say something along the lines of "all PS2 versions of X and Y are supported. There is no planned support for the PS3, PS4 or PS5 versions."

Reply:
Now, it appears I missed that this was part of the FAQ. I added it, since it's mentioned as a separated question in the Discord FAQ. This piece of text is related and should be updated:

> We support _all retail versions_ of the original _Jak and Daxter_. For more details on how your version of the games matter, see [this page](./usage/game_versions.md). <span className="reddishColor">**There is no planned support for early builds and demos.**</span>

Perhaps it can move to the question? Or the question can be dropped and it's find at the top? I'll assume the latter when updating the text.